worker/src/lib.cpp

```diff
@@ -24,9 +24,56 @@
 #include <csignal>  // sigaction()
 #include <iostream> // std::cerr, std::endl
 #include <string>
+#define LEN 10
 
 void IgnoreSignals();
 
+std::string GetUnixSocketName()
+{
+	// Grab the PID of the worker
+	char line[LEN];
+	FILE *cmd = popen("pidof -s /root/build/mediasoup/worker/out/Release/mediasoup-worker", "r");
```

Comment:
What happens if there are multiple instances of mediasoup-worker?

Reply:
This piece of code will run multiple times, and multiple sockets will be created under that directory.

Reply:
<img width="1779" alt="image" src="https://user-images.githubusercontent.com/96256664/190201740-1fbee88d-5384-4fae-8f4b-66d26efec36b.png">

Reply:
I need to look into that edge case...but @gitamirp would you know the behavior of pidof in that edge case?